gk/main.c

```diff
@@ -1107,26 +1107,6 @@ gk_setup_rss(struct gk_config *gk_conf)
 	return ret;
 }
 
-static inline unsigned short
-icmp_cksum(void *buf, unsigned int size)
```

Comment:
It makes more sense to move `icmp_cksum()` to the LLS block. And let's drop its `inline` qualifier; this function is big enough.